app/src/main/AndroidManifest.xml

```diff
@@ -101,9 +101,10 @@
             android:value=".ui.SearchActivity" />
 
         <!-- Activities -->
-        <activity android:name=".ui.ConversationList"
+        <activity android:name=".ui.ConversationActivity"
             android:launchMode="singleTop"
-            android:theme="@style/Theme.Light">
+            android:theme="@style/Theme.Light"
+            android:configChanges="orientation|screenSize">
```

Comment:
Configuration changes can lead to some unexpected results when are not handled by the system (e.g. resources are not reloaded). I'll adjust the code to make it happen automatically according to the activity lifecycle.